src/service/resources-interface.js

```diff
@@ -106,6 +127,18 @@ export class ResourcesInterface extends MutatorInterface {
    */
   schedulePass(opt_delay, opt_relayoutAll) {}
 
+  /**
+   * Enqueue, or update if already exists, a mutation task for a resource.
+   * @param {./resource.Resource} resource
+   * @param {ChangeSizeRequestDef} newRequest
+   */
+  updateOrEnqueueMutateTask(resource, newRequest) {}
+
+  /**
+   * Schedules the work pass at the latest with the specified delay.
+   */
+  schedulePassVsync() {}
```

Comment:
@dvoytenko I'm wondering why we have schedulePass & schedlePassVsync. why do we need to batch in 2 different ways?

Reply:
At this point it might be unnecessary. Initially this code likely dealt with inconsistencies with multi-browser event/rAF scheduling.

Reply:
Got it, created a clean up task to track: #25793